src/aktualizr_lite/main.cc

```diff
@@ -217,6 +210,11 @@ static int daemon_main(LiteClient &client, const bpo::variables_map &variables_m
   auto current = client.primary->getCurrent();
   LOG_INFO << "Active image is: " << current;
 
+  if (!current.MatchTarget(Uptane::Target::Unknown()) && client.dockerAppsChanged()) {
```

Comment:
@EmbeddedAndroid - here's the bug. We are calling `do_update` but on the first ever run, we won't have ever called `client.primary->updateImagesMeta()` below